doc/CODING.md

```diff
@@ -122,7 +122,7 @@ Files should start with:
 	 *
 	 * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
 	 */
-	
```

Comment:
The newline is also significant. (There should be a newline after the preamble, indicated by a tab so that it will be preserved for the codeblock.)